app/Jobs/SSL/CheckSslExpiryJob.php

```
@@ -38,79 +34,17 @@ public function handle(): void
-        try {
-            $certificate = trim($ssh->exec("sudo cat {$ssl->certificate_path}"));
-
-            if (empty($certificate) || ! str_contains($certificate, 'BEGIN CERTIFICATE')) {
-                return;
+            try {
+                $action->check($ssl, notify: true, ssh: $ssh);
+            } catch (Throwable $e) {
+                Log::warning('[SSL expiry check] Failed to check certificate', [
+                    'ssl_id' => $ssl->id,
+                    'error' => $e->getMessage(),
+                ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'handle'. (undefined)

(StaticAccess)